moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -820,11 +832,28 @@ void MotionPlanningDisplay::publishInteractiveMarkers(bool pose_update)
     }
     else
     {
+      geometry_msgs::Pose pose;
+      const auto position = query_marker_offset_position_property_->getVector();
+      pose.position.x = position.x;
+      pose.position.y = position.y;
+      pose.position.z = position.z;
+      const auto orientation = query_marker_offset_orientation_property_->getQuaternion();
```

Comment:
Going back to my previous comment.  These two new members of the class (`query_marker_offset_position_property_` and `query_marker_offset_orientation_property_`) don't seem to be used anywhere else (they aren't being passed to anything that would own their lifetime).  As a result we should deallocate them when the destructor for MotionPlanningDisplay is called.